misk/src/test/kotlin/misk/web/ActionScopedWebDispatchTest.kt

```diff
@@ -38,6 +41,20 @@ internal class ActionScopedWebDispatchTest {
     assertThat(response.body!!.string()).isEqualTo("hello Thor")
   }
 
+  @Test
+  fun actionScopeSeedDataCanBeModified() {
```

Comment:
I feel like this tests the functionality okay and is the right test class, but I'm not sure if it's the most readable.